appengine/standard/migration/taskqueue/pull-counter/README.md

```diff
@@ -0,0 +1,77 @@
+# App Engine Task Pull Queue Migration Sample
+
+This sample replaces the
+[App Engine Tasks Queue Pull Counter sample](../../../taskqueue/pull-counter)
+that used the `taskqueue` library
+available only in the App Engine Standard for Python 2.7 runtime.
+
+The sample uses a
+[Pub/Sub topic with a pull queue](https://cloud.google.com/pubsub/docs/pull)
+as recommended in
+[Migrating pull queues to Pub/Sub](https://cloud.google.com/appengine/docs/standard/python/taskqueue/pull/migrating-pull-queues)
+to perform the same functions as the earlier, Task Queue sample.
+
+The application has three functions:
+
+* Viewing the home page will display a form to specify a task name, and add
+  one more request for it. It will also show all requested tasks and their counts.
+
+* Submitting the form on the home page will queue a task request.
+
+* Requesting the page at `/_ah/start` will begin processing the queued tasks.
+  This will run without responding to the request until the request is
+  cancelled, or the request times out, which can be a significant amount
+  of time.
+
+## Setup
+
+Before you can run or deploy the sample, you will need to do the following:
+
+1. Enable the Cloud Pub/Sub API in the
+[Google Developers Console](https://console.developers.google.com/project/_/apiui/apiview/pubsub/overview).
+
+1. Set Firestore to Datastore mode in the
+[Google Developers Console](https://console.cloud.google.com/datastore/welcome).
```

Comment:
[This is what I see](https://screenshot.googleplex.com/9XDUmeNih7oojhz) when I click this link - I think it means I've already done it? A hint about what this will look like or some screenshots might help here

Reply:
I'm not going to press the upgrade button and am going to see what happens

Reply:
I've changed the wording a bit. The display in the console has changed a few times, so I don't want to get too specific in what it will look like here.